korrel8r/copyright.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
```

Comment:
unneeded file. Make sure it isn't referenced in any other script before deleting though